docker-compose.yml

```diff
@@ -1,150 +1,126 @@
-version: "3"
+x-app: &app
+  build:
+    target: development
+    context: .
+    args:
+      RUBY_VERSION: '3.1.4'
```

Comment:
Am I missing somewhere that this is used in the updated image? `RUBY_VERSION` is handled during base image builds for `ghcr.io/forem/ruby`, which happens in GHA CI, and should mostly never need re-plumbed here (I think).

Reply:
I overlooked this. You are right, I'm removing this from docker-compose.